app/forms/waste_carriers_engine/other_businesses_form.rb

```diff
@@ -4,9 +4,11 @@ module WasteCarriersEngine
   class OtherBusinessesForm < BaseForm
     attr_accessor :other_businesses
 
+    validates :other_businesses, "waste_carriers_engine/yes_no": true
+
     def initialize(transient_registration)
       super
-      self.other_businesses = @transient_registration.other_businesses
+      self.other_businesses = transient_registration.other_businesses
```

Comment:
NIT Adding a line between super and this as per other changes

```suggestion
      
      self.other_businesses = transient_registration.other_businesses
```